GetandTake/Core/Filters/LogActionFilterAttribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace GetandTake.Core.Filters;
+
+public class LogActionFilterAttribute : ActionFilterAttribute
+{
+    private readonly ILogger<LogActionFilterAttribute> _logger;
+
+    public LogActionFilterAttribute(ILogger<LogActionFilterAttribute> logger)
+    {
+        _logger = logger;
+    }
+
+    public override void OnActionExecuting(ActionExecutingContext filterContext)
+    {
+        _logger.LogInformation(nameof(OnActionExecuting), filterContext.RouteData.Values["page"]);
```

Comment:
Since `filterContext.RouteData.Values["page"]` repeats several times, it makes sense to move the logic of getting this variable into function. Also, each method in filter has its own class for `filterContext`. In this way, you can simply pass `RouteData` class into your function and get `page` value from here. Hope you got my point ;)